herd/AArch64Sem.ml

```diff
@@ -4358,6 +4468,65 @@ module Make
            let lbl_v = get_instr_label ii in
            m_fault >>| set_elr_el1 lbl_v ii
            >>! B.fault [AArch64Base.elr_el1, lbl_v]
+(* Pointer Anthentication Code `FEAT_Pauth2` *)
+        | I_PAC (key, rd, rn) ->
+            if pac then begin
```

Comment:
This is really optional, but I think you could also move the implementation of the PAC instructions that is inlined here into separate functions.